routers/analytics.js

```diff
@@ -1,6 +1,12 @@
 const router = require('express').Router();
+const { PullPlayerData } = require('../helpers');
+const { authenticateToken } = require('../middleware');
 const { GetInstances } = require('./matchmaking');
+const { readFileSync } = require('node:fs');
+const RSA = require('node-rsa');
+const cfg = require('../config.json');
 
+const EXCEPTION_LOGGING_PUBLIC_KEY = new RSA().importKey(readFileSync(cfg.exception_logging_publickey_path).toString('utf-8'), cfg.exception_logging_publickey_format);
```

Comment:
As you can see, the only encryption key even vaguely accessible from the greater internet is the public key, which should be insignificant.